packages/core/src/VNodes/ContainerNode.ts

```diff
@@ -318,19 +338,32 @@ export class ContainerNode extends AbstractNode {
      *
      * @param child
      * @param index The index at which the insertion must take place within this
-     * VNode's parent, holding marker nodes into account.
+     * VNode's parentVNode, holding marker nodes into account.
      */
     _insertAtIndex(child: VNode, index: number): void {
-        // TODO: checking `this.parent` is a hack so it will go directly to
-        // `else` when parsing.
-        if (this.parent && !this.mayContainContainers && child instanceof ContainerNode) {
-            if (!this.parent) {
+        // TODO: FIX: remove `this.parentVNode` is a hack so it will go
+        // directly to `else` when parsing. But it's false to have a div in a p
+        // when whe parse the DOM.
+        if (
+            this.parentVNode &&
+            !this.mayContainContainers &&
+            child instanceof ContainerNode &&
+            (child.tangible || child.children(ContainerNode).length)
+        ) {
+            if (!this.parentVNode) {
                 console.warn(
                     `Cannot insert a container within a ${this.name}. ` +
                         'This container having no parent, can also not be split.',
                 );
                 return;
             }
+            if (!this.breakable) {
+                console.warn(
+                    `Cannot insert a container within a ${this.name}. ` +
+                        'This container is not breakable.',
+                );
```

Comment:
Je ne comprends pas. Pourquoi soudainement on ne peut plus insérer un noeud dans un unbreakable ?

Reply:
parce que on fait this.splitAt(childAtIndex) juste après et sinon il y aura un crash...

Reply:
Ce n'est pas vrai, on ne fait un split que dans le `if(this.hasChildren()`, donc le check breakable devrait être localisé là.

Reply:
c'est pas faux Karadoc